### ISSUE TITLE
Interligne différent dans les notes et le corps du texte (Different line-spacing in notes and main text)

### ISSUE THREAD

**Issue by KumakoHachiko** [maieul/ledmac#882]
# Version

 Reledmac 2020/03/08 v2.32.5
 Reledpar 2020/03/08 v2.23.0

# TeX Engine

XeLateX

# Description
 (English follows below)

Je cherche à réaliser une édition bilingue en vis-à-vis d'un texte poétique qui demande à être très abondamment annoté (d'où le `lipsum` dans les notes, mes excuses). Pour cette raison, j'ai tenté de mettre le corps du texte en interligne double et les notes en interligne simple. Cela devrait être permis par le package `setspace`. Le manuel _LaTeX pour les sciences humaines_ (§ 23.2, p. 198) précise en effet que le changement d'interligne ne s'applique pas aux notes :

> "En général une interligne de 1,5 est recommandée pour les travaux universitaires, sauf pour les notes de bas de pages, où un interlignage simple est de mise. Cela tombe bien : le package n’applique le changement d’interlignes qu’au corps du texte. Les notes de bas de page sont donc préservées."

Néanmoins, je ne suis pas parvenu à obtenir un interligne double pour le corps du texte et simple pour les notes, même en usant de méthodes poussives comme 
```latex
\begin{spacing}{1} 
``` 
et
```latex
\end{spacing}
``` 
à l'intérieur des notes ou, moins poussif, la définition d'une nouvelle commande : 

```latex
\newcommand{\myfootnote}[1]
{\footnoteAnomk{\begin{spacing}{1}
#1
\end{spacing}}} `
 ```
Dans tous ces cas l'interligne devient simple, mais LaTeX saute automatiquement une ligne derrière le numéro de la note.
La solution est-elle sous mes yeux (et je ne la vois pas) ? Merci par avance pour tout conseil.
......................

I am currently trying to create a bilingual edition for a poetic text (with the translation on the right page). As the text in question needs to be rather heavily annotated (hence the use of `lipsum`). I wanted to display the main body of text with double-spacing, while the footnotes would remain single-spaced. This should have been enabled, or so I thought, by the `setspace` package, which is supposed to affect only the main body of text, while the footnotes should remain as they are by default, that is, single-spaced (_LaTeX pour les sciences humaines_ , § 23.2, p. 198). However, I have not been able to achieve this result, even with rather straightforward solutions, such as the use of

```latex
\begin{spacing}{1} 
``` 
and
```latex
\end{spacing}
``` 
in the footnotes, or the definition of a new command, such as :

```latex
\newcommand{\myfootnote}[1]
{\footnoteAnomk{\begin{spacing}{1}
#1
\end{spacing}}} `
 ```
In all thoses cases, I manage to have a single-spaced text in the footnotes, but LaTeX seems to automatically skip one line after the footnote number, which is not very satisfying as a result.

Is there a very simple solution that I am not seeing here ? Thank you for your help.

# Minimal Working Example

```latex 
\documentclass{book}
\usepackage{fontspec}
\usepackage{polyglossia}
\setmainlanguage{french}
\usepackage[]{setspace}

\usepackage[series={A},nocritical,noend,noeledsec,noledgroup]{reledmac}
\usepackage[shiftedpstarts]{reledpar}
\maxhnotesX{0.5\textheight}

\usepackage{lipsum}

\begin{document}

\begin{doublespacing}

\begin{pages}
\begin{Leftside}


\beginnumbering

\setstanzaindents{5,5,5,5,5,5,5,5}
\stanza
Sah' ein Knab ein Röslein stehn\footnoteAnomk{\lipsum[1]} &
Röslein auf der Heiden\footnoteAnomk{\lipsum[1]}, &
War so jung und morgenschön\footnoteAnomk{\lipsum[1]}, &
Lief er schnell, es nah zu sehn\footnoteAnomk{\lipsum[1]}, &
Sah's mit vielen Freuden\footnoteAnomk{\lipsum[1]}. &
Röslein, Röslein, Röslein rot\footnoteAnomk{\lipsum[1]}, &
Röslein auf der Heiden\footnoteAnomk{\lipsum[1]} \&

\endnumbering

\end{Leftside}

\begin{Rightside}

\beginnumbering

\setstanzaindents{5,5,5,5,5,5,5,5}
\stanza
Un enfant vit une petite rose\footnoteAmk &
Petite rose sur la lande\footnoteAmk &
Si jeune et belle en ce matin\footnoteAmk &
Qu'il accourut pour la voir de près \footnoteAmk &
Il la vit avec joie\footnoteAmk &
Rose, rose, rose rouge\footnoteAmk &
Petite rose sur la lande\footnoteAmk \&

\endnumbering
\end{Rightside}
\end{pages}
\Pages

\end{doublespacing}

\end{document}
```

**Comment by maieul** [comment]
Oui, setspace est prévu pour les notes standards de LaTeX, mais pas pour reledmac, dont il ne doit pas connaître l'existence. 
Et du coup le paragraphe que j'ai mis dans le bouquin est effectivement pas tout à fait vrai.

Je viens de regarder le code de setspace et je vois comment on pourrait faire l'équivalent pour reledmac.

Ajouter simplement cela dans votre préambule.

```

    \makeatletter
\bhooknoteX{\def\baselinestretch {\setspace@singlespace}}
\makeatother

```

Ce qui dit "au début d'une note familiar (X), remettre l'espacement standard.

**Comment by KumakoHachiko** [comment]
Ca a fonctionné, merci beaucoup. Il reste seulement le fait que l'interligne des notes est presque aléatoire (il est le plus souvent correct, mais peut tout à fait décider de sauter une ligne entre deux notes, sans prévenir). Pensez-vous qu'il existe un moyen de paramétrer cet espacement par défaut des notes ?

En vous remerciant encore.

**Comment by maieul** [comment]
heu... il y a pas de raison que cela fonctionne ainsi. Merci de me fournir un exemple pour que je  regarde ce qu'il en est.

**Comment by KumakoHachiko** [comment]
C'était un problème en lien avec les titres, en fait. Si la note était celle d'un titre de section, alors le problème se produisait.

**Comment by maieul** [comment]
heu... et c'est résolu du coup ?

**Comment by KumakoHachiko** [comment]
Oui, en enlevant la note du titre de section et en la plaçant dans le texte même (plus exactement : j'ai déplacé le contenu de la note derrière `\footnoteAnomk` dans le texte et laissé l'appel `\footnoteAmk` dans le titre, de façon à avoir l'appel de note dans le titre de section, ce qui a fonctionné).

**Comment by maieul** [comment]
hum, c'est tout de même étrange. Merci de me fournir un MWE de ce qui posait problème. Là c'est un contournement, pas une résolution du problème.
